README.md

```diff
@@ -67,6 +68,15 @@ This mechanism has three layers, in order of priority:
 
 The first two layers can be skipped by setting it’s value to `undefined`. The third layer is mandatory.
 
+## Header filtering
+
+By default msw-pact capture and serialise all request and response headers captured, in the generated pact file.
```

Comment:
It might be my own misunderstanding of MSW. Are the headers always lowercased e.g. `x-blah-blah` or can they come in other shapes?

Reply:
I think that they can be any valid header name (i.e, don't contain a colon). As an example, `Content-Type` is usually written in caps.

Some testing tools such as cypress seems to be sending only lower case headers by default, but that's just a convention in the testing lib.